queue_and_stack/dll_stack.py

```diff
@@ -6,13 +6,18 @@ class Stack:
     def __init__(self):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
-        # self.storage = ?
+        self.storage = DoublyLinkedList()
 
     def push(self, value):
-        pass
+        self.size += 1
+        self.storage.add_to_head(value)
```

Comment:
To better drive home the concept of a Stack it's best to add to `tail` that adds an item to the last node of the `linked list` and remove from `tail` that is the last node. For a Queue you add to tail and remove from head.